app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -14,82 +14,59 @@
  * limitations under the License.
  */
 
-import React, { Fragment } from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 import classNames from 'classnames/bind';
 import { useTracking } from 'react-tracking';
 import { useIntl } from 'react-intl';
-import { formatAttribute } from 'common/utils/attributeUtils';
 import { canUpdateSettings } from 'common/utils/permissions';
 import { SETTINGS_PAGE_EVENTS } from 'components/main/analytics/events';
 import {
   projectNotificationsCasesSelector,
-  projectNotificationsEnabledSelector,
+  projectNotificationsStateSelector,
   updateProjectNotificationsConfigAction,
   addProjectNotificationAction,
 } from 'controllers/project';
 import { isEmailIntegrationAvailableSelector } from 'controllers/plugins';
 import { showModalAction } from 'controllers/modal';
 import { activeProjectRoleSelector, userAccountRoleSelector } from 'controllers/user';
 import { EmptyStatePage } from 'pages/inside/projectSettingsPageContainer/content/emptyStatePage';
-import { RuleListHeader } from 'pages/common/settingsPage/ruleListHeader';
-import { RuleList } from 'pages/common/settingsPage/ruleList';
+import { Button } from 'componentLibrary/button';
+import { Checkbox } from 'componentLibrary/checkbox';
+import { updateNotificationStateAction } from 'controllers/project/actionCreators';
+import { RuleList } from '../elements/ruleList';
+import { Layout } from '../layout';
 import styles from './notifications.scss';
-import {
-  ATTRIBUTES_FIELD_KEY,
-  DEFAULT_CASE_CONFIG,
-  LAUNCH_NAMES_FIELD_KEY,
-  RECIPIENTS_FIELD_KEY,
-  SEND_CASE_FIELD_KEY,
-} from './constants';
+import { DEFAULT_CASE_CONFIG } from './constants';
 import { convertNotificationCaseForSubmission } from './utils';
 import { messages } from './messages';
+import { FieldElement } from '../elements';
+import PencilIcon from './img/pencil-inline.svg';
+import BinIcon from './img/bin-inline.svg';
+import CopyIcon from './img/copy-inline.svg';
```

Comment:
I see the same icons inside of Pattern analysis page.
Please leave just one copy and move them to the `common/img`.
Also please recheck that we don't have duplicates in `common/img`.